### DISCUSSION TITLE
How to handle  different OIDCSCOPE for sample application

### DISCUSSION THREAD

**Discussion by Sugavanam** [OpenIDC/mod_auth_openidc#1049] [General]
Hello 
I have an apache server with multiple applications protected by path & proxy it to end application. 
Example:

<VirtualHost>
Server name
Oidcredirecturi /redirect t_uri/
Oidcscope openid profile 
Oidcprovideruri 
Oidc clientsecret 
Oidcclientid

<location /abc>
AuthType openid-connect
Require valid-user
ProxyPass https://example.com/abc
ProxyPassReverse https://example.com/abc

<location/>

<location /xyz>
AuthType openid-connect
Require valid-user
ProxyPass https://example.com/xyz
ProxyPassReverse https://example.com/xyz

<location/>

<virtualHost/>

Is it possible to handle oidcscopes inside the location tag?
abc and xyz application has different scope ,how to handle multiple scopes or client id's in same vhost?

**Comment by zandbelt** [comment]
you can use `OIDCPathScope` in `<Location>` contexts, see: https://github.com/OpenIDC/mod_auth_openidc/blob/v2.4.13.2/auth_openidc.conf#L746-L751

you'll most probably also want to make sure somehow the scopes are consented to by using a `Require claim` directive, see: https://github.com/OpenIDC/mod_auth_openidc/wiki/Step-up-Authentication
